types.cc

```diff
@@ -1954,13 +1952,13 @@ struct serialize_visitor {
             *out++ = char(*v);
         }
     }
-    void operator()(const date_type_impl& t, const date_type_impl::native_type* v) {
-        if (v->empty()) {
+    void operator()(const concrete_type<db_clock::time_point>& t, const emptyable<db_clock::time_point>* v1) {
```

Comment:
I don't understand the `emptyable`.

Reply:
That is the type that is stored in the data_value.